inc/deprecated/3.8.php

```diff
@@ -278,3 +278,59 @@ function rocket_deactivate_specify_image_dimensions_with_layerslider() {
 	_deprecated_function( __FUNCTION__ . '()', '3.8', 'WP_Rocket\ThirdParty\Plugins\Slider\LayerSlider::get_subscribed_events()' );
 	remove_filter( 'rocket_buffer', 'rocket_specify_image_dimensions' );
 }
+
+/**
+ * Conflict with Age Verify: don't cache pages until the age-verified cookie is set
+ *
+ * @since 2.7
+ */
+if ( class_exists( 'Age_Verify' ) && defined( 'Age_Verify::SLUG' ) ) :
+	add_filter( 'rocket_htaccess_mod_rewrite', '__return_false', 18 );
+	add_filter( 'rocket_cache_mandatory_cookies', 'rocket_add_cache_mandatory_cookie_for_age_verify' );
+endif;
```

Comment:
You don't need to keep that part, we only want to keep the deprecated functions to make sure there is no errors